dotCMS/src/main/java/com/dotcms/graphql/exception/CustomGraphQLException.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.graphql.exception;
+
+import graphql.ErrorType;
+import graphql.GraphQLError;
+import graphql.language.SourceLocation;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
If we had to use this `CustomException` perhaps  `GraphQLException` is more suited for Base Class